tests/plugins/autolink/manual/continoustyping.md

```diff
@@ -1,6 +1,6 @@
 @bender-ui: collapsed
 @bender-tags: 4.11.0, feature, 1815
-@bender-ckeditor-plugins: clipboard, toolbar, wysiwygarea, link, autolink
+@bender-ckeditor-plugins: clipboard, toolbar, wysiwygarea, autolink
```

Comment:
The `link` plugin can also be removed from `customregex.md` and `sourcemode.md`.

Reply:
Removed `link` from plugins loaded directly by bender: https://github.com/ckeditor/ckeditor4/pull/2518/commits/fcca3bc70aed2cfb91f41de1a0f044c3a2b1c5aa